server/configuration.js

```diff
@@ -2,8 +2,8 @@ export const setEnvironment = () => {
     const productionGraphqlEndpoint = 'https://tray.io/graphql';
     const stagingGraphqlEndpoint = 'https://staging.tray.io/graphql';
 
-    const appUrlStaging = 'https://app.staging.tray.io';
-    const appUrlProd = 'https://app.tray.io';
+    const appUrlStaging = 'https://embedded.staging.tray.io';
+    const appUrlProd = 'https://embedded.tray.io';
```

Comment:
Are you sure we can modify this? I find it a bit concerning that it worked previously with `app.tray.io` as well.

Reply:
iir we have a CDN rule to redirect urls matching `app.tray.io/external/` to `embedded.tray.io/external`. That's why it works.